app/components/chart-container.hbs

```diff
@@ -0,0 +1,66 @@
+{{!-- Copyright 2020, Verizon Media. Licensed under the terms of the Apache License, Version 2.0. See accompanying LICENSE.md file for terms. --}}
+<div
+  class="chart-container"
+  {{did-insert this.fetchData @location}}
+  {{did-update this.fetchData @location}}
+  ...attributes
+>
+  <div class="chart-container__timeseries-container">
+    <TimeSeries
+      class="chart-container__timeseries"
+      @records={{this.records}}
+      @title={{t "chartTitle.time-series"}}
+      @height="200px"
+    />
+    {{#unless @isMobile}}
+      <span role="button" class="chart-container__expand-icon" {{on "click" (fn this.showModalFor "time-series")}}>
+        {{svg-jar
+          "expand-fullscreen"
+          class="m-r-3"
+          width="10px"
+          height="10px"
+        }}
+      </span>
+    {{/unless}}
+  </div>
+  {{#if this.showBarChart}}
+    <div class="chart-container__stacked-bar-container">
+      <StackedBar
+        class="chart-container__stacked-bar"
+        @records={{this.records}}
+        @title={{t "chartTitle.stacked-bar"}}
+        @width="95%"
+        @height="200px"
+      />
+      {{#unless @isMobile}}
+        <span role="button" class="chart-container__expand-icon" {{on "click" (fn this.showModalFor "stacked-bar")}}>
+          {{svg-jar
+            "expand-fullscreen"
+            class="m-r-3"
+            width="10px"
+            height="10px"
+          }}
+        </span>
+      {{/unless}}
+    </div>
+  {{/if}}
+
+  {{#if this.showModal}}
+    <div class="chart-container__modal modal is-active" ...attributes>
```

Comment:
Kind of annoying that you have to press the `x` to close the modal

Reply:
thats how the denali modal works.